p9/file.go

```diff
@@ -132,6 +132,29 @@ type File interface {
 	// On the server, FSync has a read concurrency guarantee.
 	FSync() error
 
+	// Lock locks the file. The operation as defined in 9P2000.L
+	// is fairly ambitious, being a near-direct mapping to
+	// lockf(2)/fcntl(2)-style locking, but most implementations
+	// use flock(2).
+	//
+	// Arguments are defined by the 9P2000.L standard.
+	//
+	// Pid is the pid on the client.
+	// Locktype is one of read, write, or unlock (resp. 0, 1, or 2).
+	// Flags are to block (0), meaning wait; or reclaim (1),
```

Comment:
Can you define a 

```
type LockFlags uint32

const (
  // Blocking request.
  LockFlagsBlock LockFlags = 1
  // "Reserved for future use."
  LockFlagsReclaim LockFlags = 2
)
```

and use this type in both the function decl & the Tlock message and reference it here?